index.html

```diff
@@ -33,224 +33,252 @@
-          CANOPY <span class="dot tag"> Back End dev </span>
-          <span class="dot tag">2015</span>
-        </h4>
-        <p class="title-description">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required.
-        </p>
-        <ul class="stack-group">
-          <li class="stack">html</li>
-          <li class="stack">css</li>
-          <li class="stack">javascript</li>
-        </ul>
-        <button type="button" class="btn">See Project</button>
+        <div class="image-container">
+          <img src="./images/work-image1.png" alt="tonic" class="work-img" />
+        </div>
+
+        <div class="content1">
+          <h3 class="work-title">Tonic</h3>
+          <h4 class="profession">
+            CANOPY <span class="dot tag"> Back End dev </span>
+            <span class="dot tag">2015</span>
+          </h4>
+          <p class="title-description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="stack-group">
+            <li class="stack">html</li>
+            <li class="stack">css</li>
+            <li class="stack">javascript</li>
+          </ul>
+          <button type="button" class="btn">See Project</button>
+        </div>
       </div>
 
-      <!-- Work Section 1 -->
+      <!-- Work Section 2 -->
 
       <div class="work-info2">
-        <img src="./images/work-image2.png" alt="multi" class="work-img" />
-        <h3 class="work-title">
-          Multi-Post <br />
-          Stories
-        </h3>
-        <h4>
-          CANOPY <span class="dot tag"> Back End dev </span>
-          <span class="dot tag">2015</span>
-        </h4>
-        <p class="title-description">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required.
-        </p>
-        <ul class="stack-group">
-          <li class="stack">html</li>
-          <li class="stack">css</li>
-          <li class="stack">javascript</li>
-        </ul>
-        <button type="button" class="btn">See Project</button>
+        <div class="image-container">
+          <img src="./images/work-image2.png" alt="multi" class="work-img" />
+        </div>
+        <div class="content1">
+          <h3 class="work-title">
+            Multi-Post <br />
+            Stories
+          </h3>
+          <h4>
+            CANOPY <span class="dot tag"> Back End dev </span>
+            <span class="dot tag">2015</span>
+          </h4>
+          <p class="title-description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="stack-group">
+            <li class="stack">html</li>
+            <li class="stack">css</li>
+            <li class="stack">javascript</li>
+          </ul>
+          <button type="button" class="btn">See Project</button>
+        </div>
```

Comment:
- [ ] I noticed in each card the elements are close to each other so I think you should add some margin/padding to match the Figma design as much as possible 💪🏻💪🏻 so Kindly update it in all the cards🔥

- your design
![image](https://user-images.githubusercontent.com/58553711/177900027-d1269c3d-36a5-4b1e-94e3-0c23af51fba2.png)

- Figma design 
![image](https://user-images.githubusercontent.com/58553711/177900123-1790741a-0d4d-4e90-b0fc-69c23a287c75.png)

Reply:
Correction effected.
Thanks for the review.